dataprofiler/profilers/graph_profiler.py

```diff
@@ -404,7 +404,7 @@ def _get_continuous_distribution(
                 df = pd.Series(data_as_list)
                 best_fit: str = None  # type: ignore[assignment]
                 best_mle: float = 1000
-                best_fit_properties: tuple = None  # type: ignore[assignment]
+                best_fit_attributes: tuple = None  # type: ignore[assignment]
```

Comment:
I'd recommend pairing this PR back to only the change required for cont distribution to properly calculate distribution. There are some changes (variable renaming for example) that isn't really required. Better to change only absolutely necessary things to fix the issue